### DISCUSSION TITLE
SQLitePCL.raw Maui Compatibility

### DISCUSSION THREAD

**Discussion by Orgbrat** [ericsink/SQLitePCL.raw#449] [Q&A]
The SQLite-net-pcl library latest version (1.8.116) uses a dependency of your SQLitePCLRaw.bundle_green (>= 2.0.4). The author of SQLite-net states his library is MAUI compatible since it is purely a .NET Standard 2.0 library which should just work in MAUI (.NET 6.0). Is your SQLitePCL.raw also MAUI compatible for the same reason and should everything just work?

**Comment by ericsink** [comment]
Not yet. 

There are a few things I need to do yet for net6.0 workloads tfms.

**Reply by Orgbrat** [reply]
Do you have a time frame as to when you will get the few things implemented?  And thanks Eric for the reply.

**Reply by ericsink** [reply]
It'll be done before MAUI is released.

**Reply by MisinformedDNA** [reply]
@ericsink RC1 is now available with a go-live license.

UPDATE: 

I take that back. From [.NET Blog](https://devblogs.microsoft.com/dotnet/update-on-dotnet-maui/): "We are now targeting early Q2 of 2022 for .NET MAUI GA... NET 6 RC1 is a “Go Live” release... This excludes .NET MAUI packages."

**Comment by IndianaGary** [comment]
Now that .NET 6.0 is RTM, I need a supported version of SQLite for .NET 6.0, not just for MAUI. Is there an expected time-frame for such to be available?
Thanks in advance!

**Reply by ericsink** [reply]
The current release does support .NET 6.0, as evidenced by the dependencies taken by the 6.0 versions of the EFCore and Microsoft.Data.Sqlite packages.  

If you are having a problem, please be more specific.

**Comment by IndianaGary** [comment]
Thank you for the prompt reply!!!

 

Gary

**Comment by IndianaGary** [comment]
Eric,

I'm using Frank's sqlite-net-pcl nuget package. Any reason why that won't work? It does build, I just am not able to test it yet. Probably will do so in a week or so. Too many irons in too many fires atm... :-)

Regards,
Gary

________________________________

**Comment by warpedgeoid** [comment]
Has anything changed with this issue? Seems like a pretty big deal with net6.0 being the one way forward for mobile and desktop app dev with .NET.

**Comment by ericsink** [comment]
@warpedgeoid Yes, I should have posted a comment here.  The work to support .NET 6 mobile workloads is mostly done and is being tested.  See version 2.1.0-pre20220207221914 on nuget.  If you have any problems, please open an issue.

**Comment by ericsink** [comment]
2.1.0 has been released
